server/icat/include/irods/icatDefines.h

```diff
@@ -44,26 +44,26 @@
    essentially, "does the user have ACCESS_x or better?"
 
  */
-#define ACCESS_NULL                 "null"
-#define ACCESS_EXECUTE              "execute"
-#define ACCESS_READ_ANNOTATION      "read_annotation"
-#define ACCESS_READ_SYSTEM_METADATA "read_system_metadata"
-#define ACCESS_READ_METADATA        "read_metadata"
-#define ACCESS_READ_OBJECT          "read_object"
-#define ACCESS_WRITE_ANNOTATION     "write_annotation"
-#define ACCESS_CREATE_METADATA      "create_metadata"
-#define ACCESS_MODIFY_METADATA      "modify_metadata"
-#define ACCESS_DELETE_METADATA      "delete_metadata"
-#define ACCESS_ADMINISTER_OBJECT    "administer_object"
-#define ACCESS_CREATE_OBJECT        "create_object"
-#define ACCESS_MODIFY_OBJECT        "modify_object"
-#define ACCESS_DELETE_OBJECT        "delete_object"
-#define ACCESS_CREATE_TOKEN         "create_token"
-#define ACCESS_DELETE_TOKEN         "delete_token"
-#define ACCESS_CURATE               "curate"
-#define ACCESS_OWN                  "own"
+#define ACCESS_NULL                                "null"
+#define ACCESS_EXECUTE                             "execute"
+#define ACCESS_READ_ANNOTATION                     "read_annotation"
+#define ACCESS_READ_SYSTEM_METADATA                "read_system_metadata"
+#define ACCESS_READ_METADATA                       "read_metadata"
+#define ACCESS_READ_OBJECT                         "read_object"
+#define ACCESS_WRITE_ANNOTATION                    "write_annotation"
+#define ACCESS_CREATE_METADATA                     "create_metadata"
+#define ACCESS_MODIFY_METADATA                     "modify_metadata"
+#define ACCESS_DELETE_METADATA                     "delete_metadata"
+#define ACCESS_ADMINISTER_OBJECT                   "administer_object"
+#define ACCESS_CREATE_OBJECT                       "create_object"
+#define ACCESS_MODIFY_OBJECT                       "modify_object"
+#define ACCESS_DELETE_OBJECT                       "delete_object"
+#define ACCESS_CREATE_TOKEN                        "create_token"
+#define ACCESS_DELETE_TOKEN                        "delete_token"
+#define ACCESS_CURATE                              "curate"
+#define ACCESS_OWN                                 "own"
 
-#define ACCESS_INHERIT              "inherit"
-#define ACCESS_NO_INHERIT           "noinherit"
+#define ACCESS_INHERIT                             "inherit"
+#define ACCESS_NO_INHERIT                          "noinherit"
```

Comment:
I dunno about all those spaces. :p Maybe add some `// clang-format off` for this section... Feel free to disregard.

Reply:
let it breathe!    trust the clang-format!